apps/sms/views/conversation/style/message.css

```diff
@@ -35,11 +40,15 @@
 }
 
 .message.incoming .bubble {
+  background: #39c3c2;
+}
+html[dir="ltr"] .message.incoming .bubble {
   float: left;
-
   border-radius: 0 1.5rem 1.5rem 1.5rem;
-
-  background: #39c3c2;
+}
+html[dir="rtl"] .message.incoming .bubble {
```

Comment:
Yeah, probably. Actually I spent some time reading about selector performances yesterday, and one thing that seems to stand out every time is that we shouldn't try to optimize CSS selectors until it is obvious we need to :-)